src/main/resources/templates/interest/page.html

```diff
@@ -116,5 +121,7 @@ <h4 th:text="${place.address}"></h4>
         </section>
     </div>
 </div>
+<script th:inline="javascript" src="/scripts/imageFetcher.js"></script>

```

Comment:
maybe th:inline="javascript" is redundant here?

Reply:
fixed